packages/affiliates/libs/affiliates.js

```diff
@@ -65,18 +68,16 @@ const DeployerRewards = ({ queries, empCreatorAbi, empAbi, coingecko }) => {
     }, []);
   }
 
-  // update this function to calculate a price.
-  // may want to instead get price based on collateral and collaterlization ratio
-  // returns a BigInt
-  function calculateValue(tokens, closestPrice, decimals = 18) {
-    return BigInt(tokens) * BigInt(parseFixed(closestPrice.toFixed(decimals), decimals).toString());
+  // TODO: update this function to use underlying contracts GCR and total token debt.
+  function calculateValue(tokens, closestPrice, decimals) {
+    return tokens.mul(toBN(toWei(closestPrice.toString()))).div(toBN(toWei(decimals.toString())));
```

Comment:
just curious -- why prefer BN over BigInt?

Reply:
the rest of the repo uses BigNumber & No where else do we use `BigInt`. Also, some numeric operations seemed to have less precision with `BigInt` (this was not verified just what my initial testing showed).